tools/batch/question_answering/parse_squad_results.py

```diff
@@ -14,7 +14,7 @@
 args = parser.parse_args()
 
 if args.save_path is None:
-    args.save_path = os.path.basename(os.path.dirname(args.dir)) + '.csv'
```

Comment:
if `--dir` is pointing to a directory, should the basename of the saved file come from somewhere else?

Reply:
The idea is actually to use `DIR_NAME.csv` as the file name.